app/src/main/java/org/mifos/selfserviceapp/presenters/LoginPresenter.java

```diff
@@ -166,19 +169,17 @@ private boolean isCredentialsValid(final String username, final String password)
             showEmptyInputError(context.getString(R.string.username));
             return false;
         } else if (username.length() < 5) {
-            showMinimumInputLengthNotAchievedError(username,
+            showMinimumInputLengthNotAchievedError("Username",
```

Comment:
Revert back these changes you can not use hard coded string. I have looked it was written in good manner.